GeneratorInterface/CepGenInterface/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+<use name="CepGen"/>
+<lib name="CepGenHepMC2"/>
+<use name="FWCore/Framework"/>
+<use name="GeneratorInterface/Core"/>
+<use name="clhep"/>
+<use name="hepmc"/>
+<flags CXXFLAGS="-Wno-error=missing-braces -Wno-missing-braces -Wno-non-virtual-dtor"/>
```

Comment:
Will be dropped in the next cmsdist release of CepGen (possibly as soon as 1.2.1).